docker/Dockerfile

```diff
@@ -135,7 +135,7 @@ RUN set -x \
     # Separate these due to https://git.io/fjyz6
     # Otherwise librabbitmq will install the latest amqp version,
     # violating kombu's amqp<2.0 constraint.
-    && pip install librabbitmq==1.6.1 maxminddb==1.4.1 \
+    && pip install librabbitmq==1.6.1 \
```

Comment:
The following lines above 
```
    # maxminddb
    && buildDeps="$buildDeps \
      libmaxminddb-dev \
    "\
```

are also obsolete now, right?

Reply:
Also the following below:
```
    # Fully verify that the C extension is correctly installed, it unfortunately
    # requires a full check into maxminddb.extension.Reader
    && python -c 'import maxminddb.extension; maxminddb.extension.Reader' \
```

Reply:
Don't know what I was thinking as these are still needed :D Reverted my commit and will merge once the builds pass.